### PR TITLE
adds finished exercises

### PR DESCRIPTION
**Pull request by eva-montgomery**
please review my code while you are scratching your cat!

### CODE REVIEW

**Comment by eva-montgomery** [comment]
i also added the bonus exercises

File: scripts/index.js [hunk 1]
```diff
@@ -1 +1,159 @@
...
+// Write a function which takes an array of numbers as input and 
+// returns a new array containing only the positive 
+// numbers in the given array.
+
+function posNumbers(numbers) {
+    const positiveNums = [];
+    for (let num of numbers) {
+        if (num > 0) {
+        positiveNums.push(num);
+    }
```
Comment (radishmouse) [inline scripts/index.js:11]: Great work! I do think you should indent this curly brace (and the one on the next line, though.

File: scripts/index.js [hunk 2]
```diff
@@ -1 +1,159 @@
...
+
+// Cities 2
+// Write a function which takes an array of city objects like the above problem as 
+// input and returns an array of the cities names.
+
+function cityNames(city) {
+    return (city["name"])
+}
+
+const namesCities = cities.map(cityNames);
```
Comment (radishmouse) [inline scripts/index.js:83]: Terrific!

**Review by radishmouse — COMMENTED** [review]
Great job!
👍 👍 👍

**Pull request closed without merging**